### PR TITLE
Update command line parameter documents

### PR DESCRIPTION
**Pull request by blacktear23**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/tidb/pull/36301
- Other reference link(s): https://github.com/pingcap/tidb/issues/36312

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by xhebox — APPROVED** [review]

File: command-line-flags-for-tidb-configuration.md [hunk 1]
```diff
@@ -111,6 +111,10 @@ aliases: ['/docs-cn/dev/command-line-flags-for-tidb-configuration/','/docs-cn/de
 >
 > 需谨慎使用 `*` 符号，因为它可能引入安全风险，允许来自任何 IP 的客户端自行汇报其 IP 地址。另外，它可能会导致部分直接连接 TiDB 的内部组件无法使用，例如 TiDB Dashboard。
 
+> **注意：**
+>
+> 如果使用 AWS 的 NLB 并开启 PROXY Protocol 协议，需要设置 NLB 的 `target group` 属性：`proxy_protocol_v2.client_to_server.header_place` 到 `on_first_ack`，并同时向AWS 的 Support 提工单开通此功能的支持。由于 AWS NLB 在开启 PROXY Protocol 协议的默认行为为只有在客户端发送数据之后才会发送 PROXY Protocol 的报文，同时在客户端发送数据包之前，服务器端发送的任何数据包都会在内网被丢弃，所以对于 MySQL 协议来说，客户端无法获取服务器端的握手报文，因此会一直阻塞到客户端超时。
```
Comment (TomShawn) [inline command-line-flags-for-tidb-configuration.md:116]: NBL 是负载均衡器吗？能否写全称呢？

File: command-line-flags-for-tidb-configuration.md [hunk 2]
```diff
@@ -121,6 +125,10 @@ aliases: ['/docs-cn/dev/command-line-flags-for-tidb-configuration/','/docs-cn/de
 >
 > 不要将该参数配置为 `0`。除非特殊情况，一般使用默认值即可。
 
+> **弃用：**
+>
+> 新版本中，读取 PROXY Protocol 报头的动作会在第一次读取网络数据时进行，因此为避免影响首次读取网络数据时设置的超时时间，此参数被弃用。
```
Comment (TomShawn) [inline command-line-flags-for-tidb-configuration.md:130]: 这里的「新版本」是哪个版本呢？

File: command-line-flags-for-tidb-configuration.md (hunk 1 above)
Comment (blacktear23) [reply command-line-flags-for-tidb-configuration.md]: Updated

File: command-line-flags-for-tidb-configuration.md (hunk 2 above)
Comment (blacktear23) [reply command-line-flags-for-tidb-configuration.md]: 我看应该是在 6.3 版本开始引入了对应的PR，这里已经改为 6.3.0 版本之后

File: command-line-flags-for-tidb-configuration.md [hunk 3]
```diff
@@ -111,6 +111,10 @@ aliases: ['/docs-cn/dev/command-line-flags-for-tidb-configuration/','/docs-cn/de
 >
 > 需谨慎使用 `*` 符号，因为它可能引入安全风险，允许来自任何 IP 的客户端自行汇报其 IP 地址。另外，它可能会导致部分直接连接 TiDB 的内部组件无法使用，例如 TiDB Dashboard。
 
+> **注意：**
+>
+> 如果使用 AWS 的 Network Load Balancer (简称 NLB) 并开启 PROXY Protocol 协议，需要设置 NLB 的 `target group` 属性：`proxy_protocol_v2.client_to_server.header_place` 到 `on_first_ack`，并同时向AWS 的 Support 提工单开通此功能的支持。由于 AWS NLB 在开启 PROXY Protocol 协议的默认行为为只有在客户端发送数据之后才会发送 PROXY Protocol 的报文，同时在客户端发送数据包之前，服务器端发送的任何数据包都会在内网被丢弃，所以对于 MySQL 协议来说，客户端无法获取服务器端的握手报文，因此会一直阻塞到客户端超时。
```
Comment (TomShawn) [inline command-line-flags-for-tidb-configuration.md:116]:
```suggestion
> 如果使用 AWS 的 Network Load Balancer (简称 NLB) 并开启 PROXY 协议，需要设置 NLB 的 `target group` 属性：将 `proxy_protocol_v2.client_to_server.header_place` 设为 `on_first_ack`。同时向 AWS 的 Support 提工单开通此功能的支持。由于 AWS NLB 在开启 PROXY 协议后，默认只在客户端发送数据之后才会发送 PROXY 的报文，同时在客户端发送数据包之前，服务器端发送的任何数据包都会在内网被丢弃，所以对于 MySQL 协议来说，客户端无法获取服务器端的握手报文，因此会一直阻塞到客户端超时。
```

File: command-line-flags-for-tidb-configuration.md [hunk 4]
```diff
@@ -121,6 +125,10 @@ aliases: ['/docs-cn/dev/command-line-flags-for-tidb-configuration/','/docs-cn/de
 >
 > 不要将该参数配置为 `0`。除非特殊情况，一般使用默认值即可。
 
+> **弃用：**
+>
+> 在 TiDB 6.3.0 版以后，读取 PROXY Protocol 报头的动作会在第一次读取网络数据时进行，因此为避免影响首次读取网络数据时设置的超时时间，此参数被弃用。
+
```
Comment (TomShawn) [inline command-line-flags-for-tidb-configuration.md:131]:
```suggestion
```

File: command-line-flags-for-tidb-configuration.md [hunk 5]
```diff
@@ -111,6 +111,10 @@ aliases: ['/docs-cn/dev/command-line-flags-for-tidb-configuration/','/docs-cn/de
 >
 > 需谨慎使用 `*` 符号，因为它可能引入安全风险，允许来自任何 IP 的客户端自行汇报其 IP 地址。另外，它可能会导致部分直接连接 TiDB 的内部组件无法使用，例如 TiDB Dashboard。
 
+> **注意：**
+>
+> 如果使用 AWS 的 Network Load Balancer (简称 NLB) 并开启 PROXY Protocol 协议，需要设置 NLB 的 `target group` 属性：`proxy_protocol_v2.client_to_server.header_place` 到 `on_first_ack`，并同时向AWS 的 Support 提工单开通此功能的支持。由于 AWS NLB 在开启 PROXY Protocol 协议的默认行为为只有在客户端发送数据之后才会发送 PROXY Protocol 的报文，同时在客户端发送数据包之前，服务器端发送的任何数据包都会在内网被丢弃，所以对于 MySQL 协议来说，客户端无法获取服务器端的握手报文，因此会一直阻塞到客户端超时。
+
 ## `--proxy-protocol-header-timeout`
```
Comment (TomShawn) [inline command-line-flags-for-tidb-configuration.md:119]:
```suggestion

> **注意：**
>
> - 自 v6.3.0 起，该参数被废弃。因为自 v6.3.0 起，读取 PROXY Protocol 报头的操作会在第一次读取网络数据时进行，废弃该参数可避免影响首次读取网络数据时设置的超时时间。

```

File: command-line-flags-for-tidb-configuration.md [hunk 6]
```diff
@@ -111,12 +111,20 @@ aliases: ['/docs-cn/dev/command-line-flags-for-tidb-configuration/','/docs-cn/de
...
+> 如果使用 AWS 的 Network Load Balancer (NLB) 并开启 PROXY 协议，需要设置 NLB 的 `target group` 属性：将 `proxy_protocol_v2.client_to_server.header_place` 设为 `on_first_ack`。同时向 AWS 的 Support 提工单开通此功能的支持。由于 AWS NLB 在开启 PROXY 协议后，默认只在客户端发送数据之后才会发送 PROXY 的报文，同时，在客户端发送数据包之前，服务器端发送的任何数据包都会在内网被丢弃，所以，对于 MySQL 协议来说，客户端无法获取服务器端的握手报文，因此会一直阻塞到客户端超时。
+
 ## `--proxy-protocol-header-timeout`
 
-+ PROXY Protocol 请求头读取超时时间
++ PROXY 协议请求头读取超时时间
 + 默认：5
 + 单位：秒
 
+> **警告：**
```
Comment (shichun-0415) [inline command-line-flags-for-tidb-configuration.md:124]: Shall we combine the note and the warning:
image

File: command-line-flags-for-tidb-configuration.md [hunk 7]
```diff
@@ -111,12 +111,20 @@ aliases: ['/docs-cn/dev/command-line-flags-for-tidb-configuration/','/docs-cn/de
 >
 > 需谨慎使用 `*` 符号，因为它可能引入安全风险，允许来自任何 IP 的客户端自行汇报其 IP 地址。另外，它可能会导致部分直接连接 TiDB 的内部组件无法使用，例如 TiDB Dashboard。
 
+> **注意：**
```
Comment (shichun-0415) [inline command-line-flags-for-tidb-configuration.md:114]: Shall we combine the preceding warning and this note to make the text more readable?
image

File: command-line-flags-for-tidb-configuration.md (hunk 6 above)
Comment (blacktear23) [reply command-line-flags-for-tidb-configuration.md:124]: 我看还是分开吧，毕竟是2个意思。

File: command-line-flags-for-tidb-configuration.md (hunk 7 above)
Comment (blacktear23) [reply command-line-flags-for-tidb-configuration.md:114]: 同上

File: command-line-flags-for-tidb-configuration.md (hunk 6 above)
Comment (shichun-0415) [reply command-line-flags-for-tidb-configuration.md:124]: 可能我没有说清楚，不是说把 note 和 warning 合为一句话，而是合在一个 note 下面作为两个并列的注意项，目前上面一个警告，下面一个注意，阅读体验不是很好。英文的我已经先这样改了，你看看：
image

File: command-line-flags-for-tidb-configuration.md (hunk 7 above)
Comment (shichun-0415) [reply command-line-flags-for-tidb-configuration.md:114]: 同上：
image

File: command-line-flags-for-tidb-configuration.md [hunk 8]
```diff
@@ -111,12 +111,20 @@ aliases: ['/docs-cn/dev/command-line-flags-for-tidb-configuration/','/docs-cn/de
...
 ## `--proxy-protocol-header-timeout`
 
-+ PROXY Protocol 请求头读取超时时间
++ PROXY 协议请求头读取超时时间
 + 默认：5
 + 单位：秒
 
+> **警告：**
+>
+> - 自 v6.3.0 起，该参数被废弃。因为自 v6.3.0 起，读取 PROXY 协议报头的操作会在第一次读取网络数据时进行，废弃该参数可避免影响首次读取网络数据时设置的超时时间。
```
Comment (shichun-0415) [inline command-line-flags-for-tidb-configuration.md:126]:
```suggestion
> 自 v6.3.0 起，该参数被废弃。因为自 v6.3.0 起，读取 PROXY 协议报头的操作会在第一次读取网络数据时进行，废弃该参数可避免影响首次读取网络数据时设置的超时时间。
```

**Review by xhebox — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**